env0/resource_variable_set_assignment_test.go

```diff
@@ -61,6 +61,11 @@ func TestUnitVariableSetAssignmentResource(t *testing.T) {
 				AssignmentScope:   scope,
 				AssignmentScopeId: scopeId,
 			},
+			{
```

Comment:
added this use case: this should be filtered out.
(Which mean the test outcome shouldn't change).